### PR TITLE
Bug/vspc 210

### PR DESCRIPTION
**Pull request by swetalina-prusty25**
Guidelines for Pull Requests
If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn [ ] into [x]).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

Please provide a brief description of your ticket
(you can copy the ticket if it hasn't changed)
Staff side space: there should be a link to the public side of a space
...Put description here...

Right now if I want to see how a space looks like on the public page, I have to go to the public home page and then find my way to the space I want to see. Instead there should be a direct link to a space on the staff side.

Anything else the reviewer needs to know?
... describe here ...

### CODE REVIEW

**Comment by swetalina-prusty25** [comment]
Implemented the direct link to a space on the staff side to public side space.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.html [hunk 1]
```diff
@@ -1477,6 +1477,7 @@ <h2>
             <button type="button" id="changeBgImgButton"  class="btn primary-btn" data-toggle="tooltip" title="Change Background Image" data-placement="top">
             	<i class="fas fa-pen"></i>&emsp;<i class="fas fa-image"></i></button>
             &nbsp;
+            <a th:href="@{'/exhibit/space/'+${space?.id}}" class="btn primary-btn" style="float: right; margin-right: 39rem">Public Space</a>
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.html:1480]: this doesn't look like it belongs into this ticket.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Wait, what is this PR about. You commented 
> On staff side: when clicking the drop down menu for a sequence (to edit title or description), the drop down menu says "edit Slide title" and "slide description" instead of sequence title, etc.
so changed the values to "edit sequence title" ,etc.
But that's not right, is it?

**Comment by jdamerow** [comment]
Please figure out what actually belongs into this pr and fix ticket description.

**Comment by swetalina-prusty25** [comment]
Fixed ticket description which is Staff side space: there should be a link to the public side of a space.
code changes are in spcae.html fils

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.html [hunk 2]
```diff
@@ -298,27 +298,27 @@ <h2 id="title" style="margin-right: 20px"><span style="color: var(--dark-grey)">
...
-					style="float: left; margin-right: 1%;">Save</button>
-				<button id="cancelEditTitle" type="button" class="btn primary-btn"
-					style="margin-top: 1%; margin-bottom: 1%; margin-left: .5rem;">Cancel</button>
+				<button id="submitTitle" type="button" class="btn primary-btn btn-sm"
+					style="position: absolute;margin-top: 3.1rem;margin-left: 13.3rem; ">Save</button>
+				<button id="cancelEditTitle" type="button" class="btn btn-light cancelSpaceLinkBtn btn-sm"
+					style="position: absolute;margin-left: 17.3rem;margin-top: 3.1rem;">Cancel</button>
 			</div>
 			<div class="alert alert-light" role="alert">
 				Created on <span class="date">[[${sequence.creationDate}]]</span> by
```
[the remark this answers is not in the document]
Comment (swetalina-prusty25) [reply vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.html]: sequence.html file doesn't belongs to this ticket, so removed it.
only space.html file has the required changes for this ticket.

**Comment by swetalina-prusty25** [comment]
Copied the exact version for sequence.html from develop

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
That should not be a button but an icon. And since it doesn't have anything to do with creating link or editing the space in any way, it should also not be with the buttons to create links.
image
Please read https://diging.atlassian.net/wiki/spaces/DIGING/pages/3056304129/Basic+Webpage+Design+Guidelines and let me know if there are other information that would help you with webpage layout/design.

**Pull request closed without merging**